liquibase-integration-tests/src/test/resources/changelogs/cockroachdb/complete/root.changelog.xml

```diff
@@ -134,6 +134,10 @@
     </changeSet>
 
     <changeSet id="14" author="nvoxland">
+        <comment>
+            This test also ensures that changesets are not run in a transaction. If this test is run in a transaction
+            in current cockroachdb, the insert may error.
```

Comment:
Was the test passing before your change too? I am curious what the error was, since I ran this earlier and don't remember an error.